File: DsWorkNet/Dswork.Http/NameValue.cs
using System;

namespace Dswork.Http
{
	/// <summary>
	/// 自定义NameValue
	/// </summary>
	public class NameValue
	{
		private String name;
		private String value;

		/// <summary>
		/// 构造方法
		/// </summary>
		/// <param name="name">String</param>
		/// <param name="value">String</param>
		public NameValue(String name, String value)
		{
			this.name = Convert.ToString(name);
			this.value = value;
		}

		/// <summary>
		/// Name
		/// </summary>
		public String Name
		{
			get { return name; }
			set { this.name = Convert.ToString(value); }
		}

		/// <summary>
		/// Value
		/// </summary>
		public String Value
		{
			get { return value; }
			set { this.value = value; }
		}
	}
}

File: DsWorkJava/dswork-http/dswork/http/NameValue.java
package dswork.http;

/**
 * 自定义NameValue
 * @author skey
 * @version 1.0
 */
public class NameValue
{
	private String name;
	private String value;

	/**
	 * 构造方法
	 * @param name String
	 * @param value String
	 */
	public NameValue(String name, String value)
	{
		this.name = String.valueOf(name);
		this.value = value;
	}
	
	public void setName(String name)
	{
		this.name = String.valueOf(name);
	}

	public String getName()
	{
		return this.name;
	}
	
	public void setValue(String value)
	{
		this.value = value;
	}

	public String getValue()
	{
		return this.value;
	}
}
